components/StudentResource.jsx

```diff
@@ -0,0 +1,124 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Text } from "./Text";
+import { Button } from "./Button";
+import { media } from "../utils";
+
+export const StudentResource = ({
+  titleText,
+  descriptionText,
+  buttons,
+  graphic,
+  graphicWidth,
+  graphicHeight,
+  isGraphicLeft = true,
+  ...props
+}) => (
+  <Wrapper isGraphicLeft={isGraphicLeft} {...props}>
+    <Graphic
+      src={graphic}
+      width={graphicWidth}
+      height={graphicHeight}
+      isGraphicLeft={isGraphicLeft}
+    />
+    <InfoSection isGraphicLeft={isGraphicLeft}>
+      <Title>{titleText}</Title>
+      <Text>{descriptionText}</Text>
+      <ButtonSection isGraphicLeft={isGraphicLeft}>
+        {buttons.map((button) => (
+          <Link key={button.text} href={button.link}>
+            <a>
+              <SButton backgroundColor={button.color}>{button.text}</SButton>
+            </a>
+          </Link>
+        ))}
+      </ButtonSection>
```

Comment:
I'd make this an optional prop so something like:

```
{!!isButton && (
<ButtonSection>
...
)}
```
So we can reuse even if a button is not there :')

Reply:
Can use without buttons now
![image](https://user-images.githubusercontent.com/65516245/127266871-6c31ef56-1baa-49af-bdbb-ffa3d5968ef2.png)